src/services/admin/interfaces/db-service.ts

```diff
@@ -0,0 +1,20 @@
+import { DatabaseTransactionConnection as TrxHandler } from 'slonik';
+
+import { MemberRole } from './member-role';
+
+/**
+ * Database's first layer of abstraction for Admin DB services
+ */
+export interface AdminService {
+  /**
+   * Get the role of given user
+   * @param memberId
+   * @param dbHandler Database handler
+   */
+  getMemberRoles(
+    memberId: string,
+    dbHandler: TrxHandler,
+  ): Promise<MemberRole[]>;
```

Comment:
Should it return `Role[]` directly? `MemberRole` is just a one-to-many relationship table right?